config/default/manager/manager.yaml

```diff
@@ -138,7 +138,7 @@ spec:
           capabilities:
             drop:
             - ALL
-        image: quay.io/openshift/origin-kube-rbac-proxy:4.10.0
+        image: quay.io/openshift/origin-kube-rbac-proxy:4.15.0
```

Comment:
I wonder if we want hash or version
hash is static but might be deleted (we are using hash on CNAO fwiw)
version is dynamic, not deleted by might present flakes.